e2e/wdio/platform/tests/action-bar.e2e-spec.ts

```diff
@@ -0,0 +1,67 @@
+import { ActionBarPo } from '../pages/action-bar.po';
+import { checkElArrIsClickable, checkElementTextValue } from '../../helper/assertion-helper';
+import {
+    acceptAlert,
+    click,
+    getAlertText,
+    getCSSPropertyByName,
+    mouseHoverElement,
+    scrollIntoView,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import ActionBarContent from '../fixtures/appData/action-bar-contents';
+
+describe('Action Bar Test Suite', function() {
+    const actionBarPage = new ActionBarPo();
+
+    beforeAll(() => {
+        actionBarPage.open();
+    }, 1);
+
+    describe('Main checks', function() {
+        it('should check action buttons', ()  => {
+            mouseHoverElement(actionBarPage.actionBtnArr);
+            expect(getCSSPropertyByName(actionBarPage.actionBtnArr, ActionBarContent.hoverColorAttr).value)
```

Comment:
please try to use object destructuring on import, and you will have then
```ts
expect(getCSSPropertyByName(actionBtnArr, hoverColorAttr).value).toContain(saveBtnHvrColor);
```

and code will be much more readable

Reply:
done